### ISSUE TITLE
Rewrite the Installing page

### ISSUE THREAD

**Issue by potocnyj** [VividCortex/docs#180]
Right now the installing page is rather hard to follow - for a user trying to install VividCortex for the first time, it's quite disorganized and doesn't provide a good walkthrough on what needs to be done. Let's rewrite it into a more succinct step-by-step guide that can be referenced during setup.

**Comment by potocnyj** [comment]
One thing that I think would also help here would be to split the page into multiple pages - one each for the On-Host, Off-Host and Containerized options in the Add-Host wizard.

**Comment by seaparks** [comment]
Done as part of https://github.com/VividCortex/docs/pull/249
